File: app/src/main/java/org/example/maidian/MyApp.java
package org.example.maidian;

import android.app.Application;

import org.gioneco.analytics.android.sdk.DataAPI;


/**
 * Created by zsq
 * on 2020-09-29
 */
public class MyApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        initGionecoDataAPI(this);
    }

    /**
     * 初始化埋点 SDK
     *
     * @param application Application
     */
    private void initGionecoDataAPI(Application application) {
        DataAPI.init(application);
    }
}

File: app/src/main/java/org/example/maidian/MyApp.kt
package org.example.maidian

import android.app.Application

import org.gioneco.analytics.android.sdk.BuildConfig
import org.gioneco.analytics.android.sdk.http.DataAPI


/**
 * Created by zsq
 * on 2020-09-29
 */
class MyApp : Application() {
    override fun onCreate() {
        super.onCreate()
        DataAPI.init(this)
    }
}
